tools/apidiff/repo/repo.go

```diff
@@ -172,3 +172,25 @@ func (wt WorkingTree) ListTags(pattern string) ([]string, error) {
 	sort.Strings(tags)
 	return tags, nil
 }
+
+// Pull calls "git pull upstream branch" to update local working tree.
+func (wt WorkingTree) Pull(upstream string, branch string) error {
```

Comment:
Nit, when multiple args have the same type you don't have to repeat the type name.
```go
Pull(upstream, branch string)
```

Reply:
Done